packages/widget/src/controllers/transfers/fungible-token-transfer.ts

```diff
@@ -169,31 +265,30 @@ export class FungibleTokenTransferController implements ReactiveController {
 
   onDestinationAddressChange = (address: string): void => {
     this.destinationAddress = address;
-
-    if (this.destinationAddress && this.destinationAddress.length === 0) {
```

Comment:
I think it was wrongly overwritten, the entire story https://github.com/sygmaprotocol/sygma-widget/pull/176/files which was merged to master is reverted

Reply:
I restore the changes, could you please let me know if there is anything missing? I think not but better to have two pair of eyes here